compose.env

```diff
@@ -0,0 +1,38 @@
+## OLLAMA VARS
+OLLAMA_MODEL=gemma:2b
+OLLAMA_KEEP_ALIVE=24h
+OLLAMA_HOST=0.0.0.0
+OLLAMA_PORT=11434
+
+
+## INFINITY VARS
+INFINITY_EMBEDDING_MODEL=mixedbread-ai/mxbai-embed-large-v1
+INFINITY_RERANKING_MODEL=mixedbread-ai/mxbai-rerank-xsmall-v1
+INFINITY_BATCH_SIZE=4
+INFINITY_PORT=7997
+
+## QDRANT
+QDRANT_REST=6333
+QDRANT_GRPC=6334
+
+## COGNITA_BACKEND VARS
+COGNITA_BACKEND_PORT=8000
+COGNITA_BACKEND_HOST=0.0.0.0
+OLLAMA_URL=http://host.docker.internal:${OLLAMA_PORT}
```

Comment:
for all _HOST / _URL we can use the bridge network instead of relying on host.docker.internal